src/foam/nanos/session/local_settings/LocalSetting.js

```diff
@@ -0,0 +1,22 @@
+foam.CLASS({
+  package: 'foam.nanos.session.local_settings',
+  name: 'LocalSetting',
+  properties: [
+    {
+      name: 'id',
```

Comment:
just make the name the id.  What was your intent for ID? 
If we plan to store these permanently later then each will have a User reference, and yes, then we would need an ID - but it can be an id of name, user reference and device id.  
so in general wouldn't worry about that just now.